packages/ui/src/components/molecules/DialogBox.tsx

```diff
@@ -21,24 +21,24 @@ export interface DialogBoxProps
 }
 
 const DialogBoxStyle = styled.section<DialogBoxProps>`
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
```

Comment:
Why's this changed?

Reply:
Because it is not necessary to have the children of DialogBox as centrally aligned and justified. Instead, it can be defined from the components,

Reply:
Existing pages are affected due to this. They are no longer following the figma design.